### PR TITLE
Added controllers

### PR DESCRIPTION
**Pull request by aleksandr-hrankin**

### CODE REVIEW

File: src/main/java/com/internet/shop/controller/RegistrationController.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        req.getRequestDispatcher("/WEB-INF/view/registration.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String userName = req.getParameter("userName");
+        String login = req.getParameter("login");
+        String password = req.getParameter("password");
+        String repeatPassword = req.getParameter("repeat-password");
```
Comment (dariiabutorina) [inline src/main/java/com/internet/shop/controller/RegistrationController.java:33]: You might receive the empty Strings. Let's add the check

File: src/main/java/com/internet/shop/controller/injector/InjectDataController.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        productService.create(new Product("iPhone X", 700.00));
+        productService.create(new Product("iPhone 7", 500.00));
+        productService.create(new Product("xBox 360", 480.00));
+        productService.create(new Product("Sony Playstation 4", 520.00));
+        productService.create(new Product("Samsung A50", 300.00));
+
+        User userBob = new User("Bob", "bob-the-best", "bob123");
+        User userAlice = new User("Alice", "mis-alice", "12alice12");
+        userService.create(userBob);
+        userService.create(userAlice);
```
Comment (dariiabutorina) [inline src/main/java/com/internet/shop/controller/injector/InjectDataController.java:36]:
```suggestion
        userService.create(new User("Bob", "bob-the-best", "bob123"));
        userService.create(new User("Alice", "mis-alice", "12alice12"));
```

File: src/main/java/com/internet/shop/controller/product/AddProductController.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+public class AddProductController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private ProductService productService =
+            (ProductService) injector.getInstance(ProductService.class);
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String name = req.getParameter("productName");
+        double price = Double.parseDouble(req.getParameter("productPrice"));
```
Comment (dariiabutorina) [inline src/main/java/com/internet/shop/controller/product/AddProductController.java:21]: Same check

**Review by dariiabutorina — CHANGES_REQUESTED** [review]
Not bad, let's work the issues below out

File: src/main/java/com/internet/shop/controller/cart/AddProductInCartController.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.service.interfaces.ProductService;
+import com.internet.shop.service.interfaces.ShoppingCartService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AddProductInCartController extends HttpServlet {
```
Comment (RomanDubovskyi) [inline src/main/java/com/internet/shop/controller/cart/AddProductInCartController.java]:
```suggestion
public class AddProductToCartController extends HttpServlet {
```

File: src/main/java/com/internet/shop/controller/user/DeleteUserController.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class DeleteUserController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private UserService userService = (UserService) injector.getInstance(UserService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
```
Comment (RomanDubovskyi) [inline src/main/java/com/internet/shop/controller/user/DeleteUserController.java:20]: if we add creat shoppingCart for user in registration by default, it makes sense to delete whenever we delete user

File: src/main/webapp/web.xml [hunk 6]
```diff
@@ -8,9 +8,110 @@
...
+    <servlet>
+        <servlet-name>deleteProduct</servlet-name>
+        <servlet-class>com.internet.shop.controller.product.DeleteProductController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>deleteProduct</servlet-name>
+        <url-pattern>/storage/products/delete</url-pattern>
+    </servlet-mapping>
+
+    <servlet>
```
Comment (RomanDubovskyi) [inline src/main/webapp/web.xml]: i believe this servlet should be below the  comment shopping cart

File: src/main/java/com/internet/shop/controller/user/DeleteUserController.java (hunk 5 above)
Comment (aleksandr-hrankin) [reply src/main/java/com/internet/shop/controller/user/DeleteUserController.java:20]: you're right

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**